<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="8" passed="4" failed="4" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2019-11-24T20:49:34 GMT" name="Applietool hackthon" finished-at="2019-11-24T20:53:12 GMT" duration-ms="217962">
    <groups>
    </groups>
    <test started-at="2019-11-24T20:49:34 GMT" name="Mobile portrait" finished-at="2019-11-24T20:53:12 GMT" duration-ms="217962">
      <class name="com.applitools.tests.VisualAITests">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.applitools.tests.VisualAITests@7668d560]" started-at="2019-11-24T20:49:34 GMT" name="setUp" finished-at="2019-11-24T20:49:39 GMT" duration-ms="4961" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method is-config="true" signature="setup(int, int)[pri:0, instance:com.applitools.tests.VisualAITests@7668d560]" started-at="2019-11-24T20:49:39 GMT" name="setup" finished-at="2019-11-24T20:49:42 GMT" duration-ms="3315" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[848]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[600]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method is-config="true" signature="beforeMethod(java.lang.reflect.Method)[pri:0, instance:com.applitools.tests.VisualAITests@7668d560]" started-at="2019-11-24T20:49:42 GMT" name="beforeMethod" finished-at="2019-11-24T20:49:42 GMT" duration-ms="405" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.applitools.tests.VisualAITests.loginPageUITest()]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="loginPageUITest()[pri:1, instance:com.applitools.tests.VisualAITests@7668d560]" started-at="2019-11-24T20:49:43 GMT" name="loginPageUITest" finished-at="2019-11-24T20:50:08 GMT" duration-ms="25226" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginPageUITest -->
        <test-method is-config="true" signature="beforeMethod(java.lang.reflect.Method)[pri:0, instance:com.applitools.tests.VisualAITests@7668d560]" started-at="2019-11-24T20:50:08 GMT" name="beforeMethod" finished-at="2019-11-24T20:50:08 GMT" duration-ms="2" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.applitools.tests.VisualAITests.testEmptyPassword()]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="testEmptyPassword()[pri:2, instance:com.applitools.tests.VisualAITests@7668d560]" started-at="2019-11-24T20:50:08 GMT" name="testEmptyPassword" finished-at="2019-11-24T20:50:24 GMT" duration-ms="16576" status="FAIL">
          <exception class="com.applitools.eyes.exceptions.DiffsFoundException">
            <message>
              <![CDATA[Test 'Hackathon - Test invalid data-empty password' of 'Demo App' detected differences! See details at: https://eyes.applitools.com/app/batches/00000251827672211094/00000251827672190591?accountId=NFH5dQJut06Ao5rZIFCmsQ~~]]>
            </message>
            <full-stacktrace>
              <![CDATA[com.applitools.eyes.exceptions.DiffsFoundException: Test 'Hackathon - Test invalid data-empty password' of 'Demo App' detected differences! See details at: https://eyes.applitools.com/app/batches/00000251827672211094/00000251827672190591?accountId=NFH5dQJut06Ao5rZIFCmsQ~~
at com.applitools.eyes.EyesBase.logSessionResultsAndThrowException(EyesBase.java:524)
at com.applitools.eyes.EyesBase.close(EyesBase.java:497)
at com.applitools.eyes.selenium.SeleniumEyes.close(SeleniumEyes.java:2021)
at com.applitools.eyes.selenium.Eyes.close(Eyes.java:509)
at com.applitools.eyes.selenium.Eyes.close(Eyes.java:149)
at com.applitools.tests.VisualAITests.testEmptyPassword(VisualAITests.java:129)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- com.applitools.eyes.exceptions.DiffsFoundException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testEmptyPassword -->
        <test-method is-config="true" signature="beforeMethod(java.lang.reflect.Method)[pri:0, instance:com.applitools.tests.VisualAITests@7668d560]" started-at="2019-11-24T20:50:24 GMT" name="beforeMethod" finished-at="2019-11-24T20:50:24 GMT" duration-ms="2" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.applitools.tests.VisualAITests.testEmptyUsername()]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="testEmptyUsername()[pri:2, instance:com.applitools.tests.VisualAITests@7668d560]" started-at="2019-11-24T20:50:24 GMT" name="testEmptyUsername" finished-at="2019-11-24T20:50:45 GMT" duration-ms="20845" status="FAIL">
          <exception class="com.applitools.eyes.exceptions.DiffsFoundException">
            <message>
              <![CDATA[Test 'Hackathon - Test invalid data - blank username' of 'Demo App' detected differences! See details at: https://eyes.applitools.com/app/batches/00000251827672211094/00000251827672172755?accountId=NFH5dQJut06Ao5rZIFCmsQ~~]]>
            </message>
            <full-stacktrace>
              <![CDATA[com.applitools.eyes.exceptions.DiffsFoundException: Test 'Hackathon - Test invalid data - blank username' of 'Demo App' detected differences! See details at: https://eyes.applitools.com/app/batches/00000251827672211094/00000251827672172755?accountId=NFH5dQJut06Ao5rZIFCmsQ~~
at com.applitools.eyes.EyesBase.logSessionResultsAndThrowException(EyesBase.java:524)
at com.applitools.eyes.EyesBase.close(EyesBase.java:497)
at com.applitools.eyes.selenium.SeleniumEyes.close(SeleniumEyes.java:2021)
at com.applitools.eyes.selenium.Eyes.close(Eyes.java:509)
at com.applitools.eyes.selenium.Eyes.close(Eyes.java:149)
at com.applitools.tests.VisualAITests.testEmptyUsername(VisualAITests.java:116)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- com.applitools.eyes.exceptions.DiffsFoundException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testEmptyUsername -->
        <test-method is-config="true" signature="beforeMethod(java.lang.reflect.Method)[pri:0, instance:com.applitools.tests.VisualAITests@7668d560]" started-at="2019-11-24T20:50:45 GMT" name="beforeMethod" finished-at="2019-11-24T20:50:45 GMT" duration-ms="2" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.applitools.tests.VisualAITests.testInvalidPassword()]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="testInvalidPassword()[pri:2, instance:com.applitools.tests.VisualAITests@7668d560]" started-at="2019-11-24T20:50:45 GMT" name="testInvalidPassword" finished-at="2019-11-24T20:51:37 GMT" duration-ms="51965" status="FAIL">
          <exception class="com.applitools.eyes.exceptions.DiffsFoundException">
            <message>
              <![CDATA[Test 'Hackathon - Test invalid data - password' of 'Demo App' detected differences! See details at: https://eyes.applitools.com/app/batches/00000251827672211094/00000251827672152624?accountId=NFH5dQJut06Ao5rZIFCmsQ~~]]>
            </message>
            <full-stacktrace>
              <![CDATA[com.applitools.eyes.exceptions.DiffsFoundException: Test 'Hackathon - Test invalid data - password' of 'Demo App' detected differences! See details at: https://eyes.applitools.com/app/batches/00000251827672211094/00000251827672152624?accountId=NFH5dQJut06Ao5rZIFCmsQ~~
at com.applitools.eyes.EyesBase.logSessionResultsAndThrowException(EyesBase.java:524)
at com.applitools.eyes.EyesBase.close(EyesBase.java:497)
at com.applitools.eyes.selenium.SeleniumEyes.close(SeleniumEyes.java:2021)
at com.applitools.eyes.selenium.Eyes.close(Eyes.java:509)
at com.applitools.eyes.selenium.Eyes.close(Eyes.java:149)
at com.applitools.tests.VisualAITests.testInvalidPassword(VisualAITests.java:102)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- com.applitools.eyes.exceptions.DiffsFoundException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testInvalidPassword -->
        <test-method is-config="true" signature="beforeMethod(java.lang.reflect.Method)[pri:0, instance:com.applitools.tests.VisualAITests@7668d560]" started-at="2019-11-24T20:51:37 GMT" name="beforeMethod" finished-at="2019-11-24T20:51:37 GMT" duration-ms="2" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.applitools.tests.VisualAITests.testInvalidUserName()]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="testInvalidUserName()[pri:2, instance:com.applitools.tests.VisualAITests@7668d560]" started-at="2019-11-24T20:51:37 GMT" name="testInvalidUserName" finished-at="2019-11-24T20:51:39 GMT" duration-ms="1759" status="FAIL">
          <exception class="org.openqa.selenium.NoSuchElementException">
            <message>
              <![CDATA[no such element: Unable to locate element: {"method":"css selector","selector":"#username"}
  (Session info: chrome=78.0.3904.108)
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'jyotis-MBP', ip: '2a02:c7d:da53:7900:d9f9:add9:5e5d:df72%en0', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.14.6', java.version: '12.0.2'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 78.0.3904.108, chrome: {chromedriverVersion: 77.0.3865.40 (f484704e052e0..., userDataDir: /var/folders/30/fc_n2ryn3dj...}, goog:chromeOptions: {debuggerAddress: localhost:61767}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: MAC, platformName: MAC, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify}
Session ID: b086746c9b1293f068b3571205e7d948
*** Element info: {Using=id, value=username}]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"css selector","selector":"#username"}
  (Session info: chrome=78.0.3904.108)
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'jyotis-MBP', ip: '2a02:c7d:da53:7900:d9f9:add9:5e5d:df72%en0', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.14.6', java.version: '12.0.2'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 78.0.3904.108, chrome: {chromedriverVersion: 77.0.3865.40 (f484704e052e0..., userDataDir: /var/folders/30/fc_n2ryn3dj...}, goog:chromeOptions: {debuggerAddress: localhost:61767}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: MAC, platformName: MAC, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify}
Session ID: b086746c9b1293f068b3571205e7d948
*** Element info: {Using=id, value=username}
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:481)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:323)
at org.openqa.selenium.remote.RemoteWebDriver.findElementById(RemoteWebDriver.java:372)
at org.openqa.selenium.By$ById.findElement(By.java:188)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:315)
at com.applitools.pages.LoginPage.typeUserName(LoginPage.java:23)
at com.applitools.tests.VisualAITests.testInvalidUserName(VisualAITests.java:85)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.NoSuchElementException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testInvalidUserName -->
        <test-method is-config="true" signature="beforeMethod(java.lang.reflect.Method)[pri:0, instance:com.applitools.tests.VisualAITests@7668d560]" started-at="2019-11-24T20:51:39 GMT" name="beforeMethod" finished-at="2019-11-24T20:51:39 GMT" duration-ms="2" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.applitools.tests.VisualAITests.loginAndSortTableByAmount()]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="loginAndSortTableByAmount()[pri:3, instance:com.applitools.tests.VisualAITests@7668d560]" started-at="2019-11-24T20:51:39 GMT" name="loginAndSortTableByAmount" finished-at="2019-11-24T20:52:13 GMT" duration-ms="33574" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginAndSortTableByAmount -->
        <test-method is-config="true" signature="beforeMethod(java.lang.reflect.Method)[pri:0, instance:com.applitools.tests.VisualAITests@7668d560]" started-at="2019-11-24T20:52:13 GMT" name="beforeMethod" finished-at="2019-11-24T20:52:13 GMT" duration-ms="3" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.applitools.tests.VisualAITests.testExpenseChart()]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="testExpenseChart()[pri:4, instance:com.applitools.tests.VisualAITests@7668d560]" started-at="2019-11-24T20:52:13 GMT" name="testExpenseChart" finished-at="2019-11-24T20:52:42 GMT" duration-ms="29453" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testExpenseChart -->
        <test-method is-config="true" signature="beforeMethod(java.lang.reflect.Method)[pri:0, instance:com.applitools.tests.VisualAITests@7668d560]" started-at="2019-11-24T20:52:42 GMT" name="beforeMethod" finished-at="2019-11-24T20:52:42 GMT" duration-ms="5" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.applitools.tests.VisualAITests.testDynamicContent()]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="testDynamicContent()[pri:5, instance:com.applitools.tests.VisualAITests@7668d560]" started-at="2019-11-24T20:52:42 GMT" name="testDynamicContent" finished-at="2019-11-24T20:53:11 GMT" duration-ms="29223" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testDynamicContent -->
        <test-method is-config="true" signature="afterAllTests()[pri:0, instance:com.applitools.tests.VisualAITests@7668d560]" started-at="2019-11-24T20:53:11 GMT" name="afterAllTests" finished-at="2019-11-24T20:53:12 GMT" duration-ms="161" status="FAIL">
          <exception class="java.lang.IllegalStateException">
            <message>
              <![CDATA[Client instance has been closed.]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IllegalStateException: Client instance has been closed.
at org.glassfish.jersey.internal.guava.Preconditions.checkState(Preconditions.java:169)
at org.glassfish.jersey.client.JerseyClient.checkNotClosed(JerseyClient.java:257)
at org.glassfish.jersey.client.JerseyClient.target(JerseyClient.java:279)
at org.glassfish.jersey.client.JerseyClient.target(JerseyClient.java:56)
at com.applitools.eyes.ServerConnector.closeBatch(ServerConnector.java:722)
at com.applitools.eyes.selenium.SeleniumEyes.closeBatch(SeleniumEyes.java:758)
at com.applitools.eyes.EyesRunner.deleteAllBatches(EyesRunner.java:33)
at com.applitools.eyes.EyesRunner.getAllTestResults(EyesRunner.java:25)
at com.applitools.tests.VisualAITests.afterAllTests(VisualAITests.java:178)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:340)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:294)
at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:217)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:130)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IllegalStateException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterAllTests -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.applitools.tests.VisualAITests@7668d560]" started-at="2019-11-24T20:53:12 GMT" name="tearDown" finished-at="2019-11-24T20:53:12 GMT" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.applitools.tests.VisualAITests -->
    </test> <!-- Mobile portrait -->
  </suite> <!-- Applietool hackthon -->
</testng-results>
